sssom/cli.py

```diff
@@ -73,18 +70,7 @@
 @click.option("-v", "--verbose", count=True)
 @click.option("-q", "--quiet")
 def main(verbose: int, quiet: bool):
-    """Main
```

Comment:
Again - it makes no sense to write docs for CLIs like this since click autogenerates them

Reply:
I am happy for all docstrings to be removed from cli.py